pkg/appmanager/resourceConfig.go

```diff
@@ -2062,6 +2070,30 @@ func (appMgr *Manager) handleRouteTls(
 	if urlRewriteRule != nil || len(appRootRules) != 0 {
 		rc.MergeRules(appMgr.mergedRulesMap)
 	}
+
+	// Add whitelist conditions to every rule (to prevent access)
+	var whitelistSourceRanges []string
+	if sourceRange, ok := route.ObjectMeta.Annotations[f5VsWhitelistSourceRangeAnnotation]; ok {
+		whitelistSourceRanges = parseWhitelistSourceRangeAnnotations(sourceRange)
+	}
+	if len(whitelistSourceRanges) > 0 {
+		for _, pol := range rc.Policies {
```

Comment:
Can we turn this block into a common function as well?

Reply:
This block is slightly unique to this function.